src/main/java/com/mealmentor/enterprise/MealMentorController.java

```diff
@@ -16,14 +19,17 @@
 import java.util.List;
 
 import com.mealmentor.enterprise.dto.Recipe;
-import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.ModelAndView;
 
 @Controller
 public class MealMentorController {
 
+
+    Logger logger = LoggerFactory.getLogger(MealMentorController.class);
+
```

Comment:
Added a logger. If you look at the catch blocks, you can see its implementation. Logging is a better way to log & keep track of errors than System.out.println().

Reply:
Definitely.  Good idea.